web/src/ProviderEditPage.js

```diff
@@ -223,6 +223,8 @@ class ProviderEditPage extends React.Component {
                   this.updateProviderField("subType", "yi-lightning");
                 } else if (value === "Silicon Flow") {
                   this.updateProviderField("subType", "deepseek-ai/DeepSeek-R1");
+                } else if (value === "GitHub") {
+                  this.updateProviderField("subType", "gpt-4o");

```

Comment:
There are many models, not just "gpt-4o"

Reply:
Isn't this setting a default value? Do you mean to add more models in setting.js?